blocks/api/registration.js

```diff
@@ -16,9 +16,34 @@ import { applyFilters } from '@wordpress/hooks';
 import { getCategories } from './categories';
 
 /**
- * Block settings keyed by block name.
+ * Defined behavior of a block type.
  *
- * @type {Object}
+ * @typedef {WPBlockType}
```

Comment:
We should just use flow or typescript :trollface:

Reply:
Yes, that would even more useful :)